python/pyarrow/_flight.pyx

```diff
@@ -118,12 +118,18 @@ cdef class FlightCallOptions(_Weakrefable):
         write_options : pyarrow.ipc.IpcWriteOptions, optional
             IPC write options. The default options can be controlled
             by environment variables (see pyarrow.ipc).
-
+        headers : vector[pair[c_string, c_string]], optional
```

Comment:
This type hint should use Python conventions (e.g. `List[Tuple[str, str]]`)

Reply:
Addressed.